src/components/style.css

```diff
@@ -2,20 +2,296 @@
 @tailwind components;
 @tailwind utilities;
 
-body{
+body {
     background: #303030;
+    height: 100%;
+    width: 100%;
+    margin-top: 0px;
+    margin-left: 0px;
+    margin-right: 0px;
+    background-image: var(--body-bg);
+    font-family: Helvetica, sans-serif;
 }
+#email{}
 
-.email-address{
+.roomName{
+    margin: 0;
+    text-align: center;
+    color: #fff;
+    font-weight: bold;
+    flex-shrink: 0;
+}
+.chat{
+    overflow-y: auto;
+    flex: 1;
+}
+
+/* width */
+.chat::-webkit-scrollbar {
+    width: 10px;
+  }
+  
+  /* Track */
+  .chat::-webkit-scrollbar-track {
+    background: #303030;
```

Comment:
lägg in alla färger i variablar så att dem är enklare att nå i alla css filer, och även bätre practise att dela upp css fil per komponent.